src/libraries/Common/src/Interop/Unix/System.Native/Interop.SysLog.cs

```diff
@@ -11,36 +11,9 @@ internal static partial class Sys
         internal enum SysLogPriority : int
         {
             // Priorities
-            LOG_EMERG       = 0,        /* system is unusable */
-            LOG_ALERT       = 1,        /* action must be taken immediately */
-            LOG_CRIT        = 2,        /* critical conditions */
-            LOG_ERR         = 3,        /* error conditions */
-            LOG_WARNING     = 4,        /* warning conditions */
-            LOG_NOTICE      = 5,        /* normal but significant condition */
-            LOG_INFO        = 6,        /* informational */
             LOG_DEBUG       = 7,        /* debug-level messages */
             // Facilities
-            LOG_KERN        = (0<<3),   /* kernel messages */
             LOG_USER        = (1<<3),   /* random user-level messages */
```

Comment:
Since we are always logging as `LOG_USER` and it is likely going to always stay that way, I am wondering why we need this at all. Can this bit be flipped in the PAL?

Reply:
It seems like we can simplify `DebugProvider.Unix.cs` as:

```diff
- Interop.Sys.SysLog(Interop.Sys.SysLogPriority.LOG_USER | Interop.Sys.SysLogPriority.LOG_DEBUG, "%s", message);
+ Interop.Sys.SysLog(Interop.Sys.SysLogPriority.LOG_DEBUG, "%s", message);
```

and remove `LOG_USER` (and `// Facilities`). Not sure about any potential downside?

Reply:
Yes, that's what I meant.